src/WebJobs.Extensions.DurableTask/EventSourceListener.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class EventSourceListener : EventListener
+    {
+        private readonly LinuxAppServiceLogger logger;
+
+        public EventSourceListener(LinuxAppServiceLogger logger)
+        {
+            this.logger = logger;
+        }
+
+        protected override void OnEventSourceCreated(EventSource eventSource)
+        {
+            if ((eventSource.Name == "DurableTask-Core"
+                  && eventSource.Guid != new Guid("7DA4779A-152E-44A2-A6F2-F80D991A5BEE")) ||
```

Comment:
Let's add a comment explaining that 7DA4... is the old, legacy DurableTask-Core provider and we want to explicitly ignore that.